bin/fuel-core/src/cli/run.rs

```diff
@@ -210,10 +211,14 @@ pub struct Command {
     pub min_da_gas_price: u64,
 
     /// P component of DA gas price calculation
```

Comment:
All I know is, never ask a guy about his D component... 😄
For reference: https://en.wikipedia.org/wiki/Proportional%E2%80%93integral%E2%80%93derivative_controller

Reply:
P (proportional) effects how fast it corrects profit "error".  In this case, a larger P value results in a slower change. Essentially it's `p_component = profit / P`

D (Derivative) effects how it responds to the slope (derivative) of the profit. So if it's quickly changing, but the error isn't big, then it should cancel out and potentially lower the gas price. This is so the profit doesn't oscillate, but instead dampens out.

(we don't have the "I" (integral) value, which I think of as the Inertia value, where if it has been a value for a long time, it resists change)

Reply:
"I" gives more than just inertia though. If you just have P and D you may get systematic deviations from the target for some systems. "I" will help break that down, since it is sensitive to longer deviations whereas P and D are "memoryless". Don't think that matters for our system though.